engines/stark/console.cpp

```diff
@@ -599,12 +600,13 @@ bool Console::Cmd_ChangeChapter(int argc, const char **argv) {
 		debugPrintf("changeChapter [value]\n");
 		return true;
 	}
-
-	long value = strtol(argv[1], 0, 10);
-	if (value >= 0 && value <= INT_MAX)
-		StarkGlobal->setCurrentChapter((int32) value);
-	else
-		debugPrintf("Invalid chapter\n");
+	
+	long value = strtol(argv[1], nullptr, 10);
```

Comment:
Actually the check is not working because `strtol` is not called correctly. I can say `changeChapter chapter1` and get no error.